content/intro-to-storybook/react/es/screen.md

```diff
@@ -11,146 +11,410 @@ En este capítulo continuaremos aumentando la sofisticación combinando componen
 
-// Un mock super simple de un store de redux
-const store = {
-  getState: () => {
-    return {
-      tasks: defaultTasks,
-    };
+```diff:title=src/components/InboxScreen.stories.js
+import React from 'react';
+import InboxScreen from './InboxScreen';
+import store from '../lib/store';
+import { rest } from 'msw';
+import { MockedState } from './TaskList.stories';
+import { Provider } from 'react-redux';
++ import {
++  fireEvent,
++  within,
++  waitFor,
++  waitForElementToBeRemoved
++ } from '@storybook/testing-library';
+export default {
+  component: InboxScreen,
+  title: 'InboxScreen',
+  decorators: [(story) => <Provider store={store}>{story()}</Provider>],
+};
+const Template = () => <InboxScreen />;
+export const Default = Template.bind({});
+Default.parameters = {
+  msw: {
+    handlers: [
+      rest.get(
+        'https://jsonplaceholder.typicode.com/todos?userId=1',
+        (req, res, ctx) => {
+          return res(ctx.json(MockedState.tasks));
+        }
+      ),
+    ],
+  },
+};
++ Default.play = async ({ canvasElement }) => {
++   const canvas = within(canvasElement);
++   // Espera a que el componente pase del estado de carga
++   await waitForElementToBeRemoved(await canvas.findByTestId('loading'));
++   // Espera a que el componente se actualice en función del store
++   await waitFor(async () => {
++     // Simula anclando la primera tarea
++     await fireEvent.click(canvas.getByLabelText('pinTask-1'));
++     // Simula anclando la tercera tarea
++     await fireEvent.click(canvas.getByLabelText('pinTask-3'));
++   });
++ };
+export const Error = Template.bind({});
+Error.parameters = {
+  msw: {
+    handlers: [
+      rest.get(
+        'https://jsonplaceholder.typicode.com/todos?userId=1',
+        (req, res, ctx) => {
+          return res(ctx.status(403));
+        }
+       ),
+    ],
   },
-  subscribe: () => 0,
-  dispatch: action('dispatch'),
 };
-
-storiesOf('InboxScreen', module)
-  .addDecorator((story) => <Provider store={store}>{story()}</Provider>)
-  .add('default', () => <PureInboxScreen />)
-  .add('error', () => <PureInboxScreen error="Something" />);
 ```
 
-Existen enfoques similares para proporcionar un contexto simulado para otras bibliotecas de datos, tales como [Apollo](https://www.npmjs.com/package/apollo-storybook-decorator), [Relay](https://github.com/orta/react-storybooks-relay-container) y algunas otras.
-
-Recorrer los estados en Storybook hace que sea fácil comprobar que lo hemos hecho correctamente:
-
-<video autoPlay muted playsInline loop >
+Revisa la historia `Default`. Haz click en el panel de `Interactions` para ver la lista de interacciones dentro de la función play de la historia.
 
+<video autoPlay muted playsInline loop>
   <source
-    src="/intro-to-storybook/finished-inboxscreen-states.mp4"
+    src="/intro-to-storybook/storybook-interactive-stories-play-function-6-4.mp4"
     type="video/mp4"
   />
 </video>
 
+### Automatizar pruebas con el corredor de pruebas
```

Comment:
@lmac-1, one small item here, if you could update the heading and the text below to reference "test runner" instead of "corridor de pruebas" I'd appreciate it. The reasoning behind this is that, in this case, we're referring to a package and not functionality, and the same rules apply as with, let's say, for example, GitHub, React, or Storybook, as they shouldn't be translated. I know this may look a bit weird and kinda breaks the wording by introducing a foreign word in the middle of the text with the choice of naming and for that, I'm sorry for it

Reply:
That makes perfect sense. I'll make the update.

Reply:
Fixed in [this commit](https://github.com/lmac-1/learnstorybook.com/commit/ea88b2160618bdcd89cb2815ce37cefd9501ae76) ✅